### PR TITLE
Grafana config in terraform

### PR DESCRIPTION
**Pull request by mattkaem**
* added sql database creation for grafana to persist settings
* added possibility to expose grafana externally via ingress

### CODE REVIEW

File: terraform/modules/cloud_sql/variables.tf [hunk 1]
```diff
@@ -61,9 +61,14 @@ variable "sql_db_user_name" {
   description = "The name of the user. Changing this forces a new resource to be created."
 }
 
-variable "sql_database_name" {
+variable "sql_hono_database_name" {
   type        = string
-  description = "The name of the database in the Cloud SQL instance. This does not include the project ID or instance name."
+  description = "The name of the hono database in the Cloud SQL instance. This does not include the project ID or instance name."
```
Comment (julian-sotec) [inline terraform/modules/cloud_sql/variables.tf:66]: I would suggest to change it to this (There are other places too - if we decide to change - we should change all places)
> ... the Google Project ID ...

File: terraform/environment-template/infrastructure/outputs.tf [hunk 2]
```diff
@@ -87,6 +102,11 @@ output "cert_manager_sa_key_file" {
 }
 
 output "cert_manager_issuer_project_id" {
-  value        = module.infrastructure.cert_manager_issuer_project_id
+  value       = module.infrastructure.cert_manager_issuer_project_id
   description = "Project ID in which the Cloud DNS zone to manage the DNS entries is located."
 }
+
+output "grafana_expose_externally" {
```
[the remark this answers is not in the document]
Comment (mattkaem) [reply terraform/environment-template/infrastructure/outputs.tf:109]: For the mqtt / http adapter we don't use an ingress but rather only a loadbalancer which is configured a bit differently. Therefor we definitively need this variable to activate the deployment of the ingress.

File: terraform/infrastructure/outputs.tf [hunk 3]
```diff
@@ -1,87 +1,107 @@
...
 output "http_static_ip" {
   value       = module.networking.http_static_ip
-  description = "Output of the mqtt static ip address"
+  description = "Output of the mqtt static ip address."
 }
 
 output "mqtt_static_ip" {
   value       = module.networking.mqtt_static_ip
-  description = "Output of the mqtt static ip address"
+  description = "Output of the mqtt static ip address."
```
[the remark this answers is not in the document]
Comment (mattkaem) [reply terraform/infrastructure/outputs.tf]: Makes sense, corrected

File: terraform/infrastructure/outputs.tf [hunk 4]
```diff
@@ -1,87 +1,107 @@
 output "gke_cluster_name" {
   value       = module.gke.gke_cluster_name
-  description = "Name of the GKE Cluster"
+  description = "Name of the GKE Cluster."
 }
 
 output "http_static_ip" {
   value       = module.networking.http_static_ip
-  description = "Output of the mqtt static ip address"
+  description = "Output of the mqtt static ip address."
```
[the remark this answers is not in the document]
Comment (mattkaem) [reply terraform/infrastructure/outputs.tf]: Makes sense, corrected

File: terraform/infrastructure/outputs.tf [hunk 5]
```diff
@@ -1,87 +1,107 @@
 output "gke_cluster_name" {
   value       = module.gke.gke_cluster_name
-  description = "Name of the GKE Cluster"
+  description = "Name of the GKE Cluster."
```
[the remark this answers is not in the document]
Comment (mattkaem) [reply terraform/infrastructure/outputs.tf]: Definitively, corrected

File: terraform/modules/cloud_sql/variables.tf (hunk 1 above)
Comment (mattkaem) [reply terraform/modules/cloud_sql/variables.tf]: Done

File: terraform/environment-template/infrastructure/outputs.tf (hunk 2 above)
[the remark this answers is not in the document]
Comment (julian-sotec) [reply terraform/environment-template/infrastructure/outputs.tf:109]: okay, makes sense to me now.

**Review by julian-sotec — APPROVED** [review]
Looks good to me now - structured objects will be implemented in separat issue / pr.

**Pull request merged**